suites/pacific/cephadm/tier-1_cephadm_cluster_view.yaml

```diff
@@ -0,0 +1,87 @@
+#===============================================================================================
+#--------------------------------------
+#----- Ceph code changes in 5.1 -------
```

Comment:
This tc is not specific to 5.1, in that case please modify accordingly

Reply:
Updated now.